### ISSUE TITLE
No chapters showing on MangasNoSekai extension

### ISSUE THREAD

**Issue by IMX-Otaku** [KotatsuApp/kotatsu-parsers#950]
### Source information

MangasNoSekai

### Steps to reproduce

1. Go to MangasNoSekai extension and open it
2. Go to any title
3. You will see that the read button says "No chapters available"

### Kotatsu version

7.3

### Android version

Android 14

### Other details

The extension settings are default ones.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.

**Comment by davvarrr** [comment]
it's partially fixed in https://github.com/KotatsuApp/kotatsu-parsers/commit/50194df24df27303d77cc48537fdf2b076a5296d, we only manage to get the 1st chapter page (so the latest releases) but the others are blocked by a rather annoying script.
